sitri/providers/contrib/system.py

```diff
@@ -21,15 +21,20 @@ def prefixize(self, key: str) -> str:
 
         :param key: varname without prefix
         """
-        return f"{self._prefix}_{key.upper()}"
+        if self._prefix:
```

Comment:
\_\_init\_\_ typing not allow None for prefix arg. If get without prefix required - change initialization method